src/main/java/frc/robot/subsystems/drivetrain/commands/DriveSlowAccel.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+
+import static frc.robot.Constants.SwerveDrive.ROTATION_MULTIPLIER;
+import static frc.robot.Constants.SwerveDrive.VELOCITY_MULTIPLIER;
+
+public class DriveSlowAccel extends CommandBase {
```

Comment:
extend from HolonomicDrive and use his functions!

Reply:
Also, I don't think you need this command, we already have the RAMP_RATE to handle it (@adamv17 ?)